webui/src/lib/components/repository/ObjectsDiff.jsx

```diff
@@ -14,6 +14,7 @@ export const ObjectsDiff = ({diffType, repoId, leftRef, rightRef, path}) => {
     let right;
     switch (diffType) {
         case 'changed':
+        case 'conflict':
```

Comment:
Can you please document the decision for how content-based diff will behave in case of a conflict? like a summary of the discussion you had over the github ticket.